src/test/resources/CCD_MedicalEquipment_With_Organizer_Procedure_Supply.xml

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@taralecw 

Please move section XML to file C-CDA_2-1_remaning_section_test.xml as we will be using it for remaining sections.